code-challenges.js

```diff
@@ -10,30 +10,81 @@
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp = 42
-// var temp = 350
-// var temp = 212
-
+var temp = 350
+var temp = 212
+
+//***********************//
+var tempA = 42
+var tempB = 350
+var tempC = 212
+var tempS = [tempA, tempB, tempC]
+    for(i=0; i < tempS.length; i++) {
+      if(temp > 212){
+        console.log(`${temp[i]} is above boiling point`);
+      } else if (temp === 212){
+        console.log(`${temp[i]} is at boiling point`);
+      } else {
+        console.log(`${temp[i]} is below boiling point`);
+      }
+    }
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+// var myNumbers1 = [3, 7, 0, 36, -9]
+// var myNumbers2 = [8, -7, 42, 9, 13]
+// console.log(myNumbers1.length + myNumbers2.length)
+
+//******//
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 42, 9, 13]
+var sumNum = [];
+
+for(let i=0; i < myNumbers1.length; i++){
+  sumNum.push(myNumbers1[i]);
+}
+for(let i=0; i < myNumbers2.length; i++){
+  sumNum.push(myNumbers2[i]);
+}
+console.log(sumNum.length);
```

Comment:
Interesting approach. You got the right answer, but there is an easier way using built in methods.

https://github.com/learn-academy-2021-echo/Syllabus/blob/main/javascript/arrays.md#strings-to-arrays-and-back-again